art/attacks/evasion/projected_gradient_descent/projected_gradient_descent_numpy.py

```diff
@@ -136,6 +135,7 @@ def generate(self, x, y=None, **kwargs):
             self.eps = np.round(self.norm_dist.rvs(1)[0], 10)
             self.eps_step = ratio * self.eps
 
+        mask = kwargs.get("mask")
```

Comment:
Why did you move this line here? If it is to have it at the beginning, why not put it right after the docstring?

Reply:
In the numpy version, this line is put within the `if` clause and is referenced in the `else` clause, so should be corrected by putting it before the `if` clause, could be right after the docstring.